CHANGELOG.md

```diff
@@ -11,6 +11,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+### Fixed
+
+- `opentelemetry-instrumentation` Catch `ModuleNotFoundError` when the library is not installed
+  and prevent exception from bubbling up
```

Comment:
I personally would change the second line here to better state that the exception is not going to the caller but instead just getting its traceback logged.

```
and prevent exception from bubbling up
```

to

```
and log as debug instead of exception
```